docs/guides/fetch-data/01-read-profile-data.md

```diff
@@ -0,0 +1,526 @@
+---
+sidebar_label: 'Read Profile Data'
+sidebar_position: 3.1
+---
+
+# Read Universal Profile Data
+
+In this guide, we will learn how to:
+
+- call a Universal Provile address
+- read data from a Universal Profile
+
+![Universal Profile example on universalprofile.cloud](./img/example-up.png)
+
+We will use:
+
+- [web3.js](https://web3js.readthedocs.io/en/v1.7.0/) for utility as well as connecting with the LUKSO network
+- [erc725.js](https://docs.lukso.tech/tools/erc725js/getting-started/) library to check the interface of an profile.
```

Comment:
can you use a relative link (`../../`) instead of using the public URL.

This allows us to keep track of broken links and also make this more robust